catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java

```diff
@@ -72,6 +73,8 @@
 import ddf.catalog.source.InternalIngestException;
 import ddf.catalog.source.SourceUnavailableException;
 import ddf.catalog.util.impl.Requests;
+import ddf.security.SecurityConstants;
+import ddf.security.Subject;
 
 public class CreateOperations {
```

Comment:
Don't know if we're planning on keeping these classes moving forward or if they are just steps in the overall refactoring, but if we plan to keep them, we really should add good JavaDoc comments to make sure we're clear on their scope and responsibilities.